src/main/java/GuiFileChooser.java

```diff
@@ -21,7 +22,8 @@ public File getDirectoryFromUser() {
         JFrame jFrame = new JFrame("FileChooserDemo");
         jFrame.setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
         JFileChooser fc = new JFileChooser();
-        fc.setFileSelectionMode(JFileChooser.DIRECTORIES_ONLY);
+        fc.setFileFilter(new FileNameExtensionFilter("SB2 Files", "sb2"));
```

Comment:
I think that as SB2 files is a the format being shown by the filter that changing it as suggested would make it more confusing.  Instead we can look into a popup box to prompt the user or some other form of instruction.